colcon-build-and-test/action.yaml

```diff
@@ -90,6 +90,4 @@ runs:
         path: |
           ./build
           ./install
-          ./lcov
-          ./coveragepy
```

Comment:
So do you mean `path` should be exactly the same on the save-side and the load-side?

Reply:
Yes. After removed these two lines, cache works correctly.